src/core/Akka/Actor/ActorCell.cs

```diff
@@ -216,7 +236,7 @@ public void Init(bool sendSupervise, MailboxType mailboxType)
             }
 
             SwapMailbox(mailbox);
-            Mailbox.SetActor(this);
+            Mailbox!.SetActor(this);
```

Comment:
`Mailbox` can't be `null` by the time anyone calls this method - or else there's been a catastrophic failure.